Tests/MapboxDirectionsTests/RouteOptionsTests.swift

```diff
@@ -31,6 +31,50 @@ class RouteOptionsTests: XCTestCase {
         XCTAssertEqual(unarchivedOptions.roadClassesToAvoid, options.roadClassesToAvoid)
     }
     
+    func testCodingWithRawCodingKeys() {
+        let routeOptionsJSON: [String: Any?] = [
+            "waypoints": [
+                [
+                "location": [-77.036500000000004, 38.8977],
+                "name": "White House",
+                ]
+            ],
+            "profile": "mapbox/driving-traffic",
+            "steps": true,
+            "geometries": "polyline",
+            "overview": "simplified",
+            "annotations": ["congestion"],
+            "language": "en_US",
+            "voice_instructions": true,
+            "voice_units": "imperial",
+            "banner_instructions": true,
+            "continue_straight": true,
+            "alternatives": false,
+            "roundabout_exits": true,
+            "exclude": ["toll"],
+            "enable_refresh": false
+        ]
+        
+        let routeOptionsData = try! JSONSerialization.data(withJSONObject: routeOptionsJSON, options: [])
+        var routeOptions: RouteOptions?
+        XCTAssertNoThrow(routeOptions = try JSONDecoder().decode(RouteOptions.self, from: routeOptionsData))
```

Comment:
mb it's easier to use `try!` there, so all the asserts can use routeOptions without unwrapping an optional?

Reply:
I'll do that. Thank you for the suggestion!